addons/mail/controllers/thread.py

```diff
@@ -163,3 +163,25 @@ def mail_message_update_content(self, message_id, body, attachment_ids, attachme
 
     def _is_message_editable(self, message, **kwargs):
         return message.sudo().is_current_user_or_guest_author or request.env.user._is_admin()
+
+    @http.route("/mail/thread/get_followers", methods=["POST"], type="jsonrpc", auth="public")
+    def mail_thread_get_followers(self, id, thread_id, thread_model, limit=20, offset=0):
+        domain = [
+            ("res_id", "=", thread_id),
+            ("res_model", "=", thread_model),
+        ]
+        data = request.env["mail.followers"].search(
+                domain + [("partner_id", "!=", request.env.user.partner_id.id)], offset=offset, limit=limit + 1, order="name ASC"
+            )
+        followerListView = {
+                "id": id,
+                "followersFullyLoaded": len(data) <= limit,
+        }
+        if not offset:
+            self_follower = request.env["mail.followers"].search(domain + [("partner_id", "=", request.env.user.partner_id.id)],)
```

Comment:
IMO this field should be related to `thread.selfFollower`. So, the one in the `FollowerListView` should be updated whenever the one in the `thread` gets updated.